Dockerfile

```diff
@@ -25,49 +23,19 @@ RUN apt-get update && \
   octave \
   gnuplot \
   ghostscript \
+  texinfo \
   zip \
+  fonts-freefont-otf \
   && \
   apt-get clean && rm -rf /var/lib/apt/lists/*
 
 RUN pip --no-cache --quiet install --upgrade \
-  pip~=21.3 \
+  pip \
   setuptools \
   wheel
 
 USER $NB_UID
 
-# jupyter customizations
-RUN conda install --quiet --yes \
-  'jupyterlab-git' \
-  # https://github.com/jupyterlab/jupyterlab-latex/
-  'jupyterlab_latex' \
-  'octave_kernel' \
-  'texinfo' \
-  'watchdog[watchmedo]' \
-  && \
-  conda clean --all -f -y && \
-  # Voila installation https://github.com/voila-dashboards/voila
-  pip install --no-cache voila && \
-  jupyter serverextension enable voila --sys-prefix && \
-  # lab extensions
-  # https://github.com/jupyter-widgets/ipywidgets/tree/master/packages/jupyterlab-manager
-  jupyter labextension install @jupyter-widgets/jupyterlab-manager --no-build && \
```

Comment:
what about these extensions? when you say "brought back" to what do you refer? Notice that some of these extesnsions have a part that require node build (i.e. not only python packages)

Reply:
I actually skipped this one.
With version `3.X` it is no longer require to build extensions. You can simply pip install them.

Reply:
ok. that is nice. That means that we can install extensions from the UI?

In any case, I think we should at least pre-install the extensions we had in previous versions...  or should we drop it and add a note saying that "you can install it yourself"?

@KZzizzle , what do you think?